judge/contest_format/ecoo.py

```diff
@@ -125,3 +125,25 @@ def display_participation_result(self, participation):
             points=floatformat(participation.score, -self.contest.points_precision),
             cumtime=nice_repr(timedelta(seconds=participation.cumtime), 'noday') if self.config['cumtime'] else '',
         )
+
+    def get_short_form_display(self):
+        yield _('The score on your **last** non-CE submission for each problem will be used.')
+
+        first_ac_bonus = self.config['first_ac_bonus']
+        if first_ac_bonus:
+            yield _(
+                'There is a **%d bonus** for fully solving on your first non-CE submission.',
+            ) % first_ac_bonus
+
+        time_bonus = self.config['time_bonus']
+        if time_bonus:
+            yield ungettext(
+                'For every **1 minute** you submit before the end of your window, there will be a **1** point bonus.',
```

Comment:
Why do you hardcode the `1` here but no where else?

Reply:
fixed